simde/x86/avx512vl.h

```diff
@@ -352,6 +352,110 @@ simde_mm256_maskz_abs_epi64(simde__mmask8 k, simde__m256i a) {
 #define _mm256_maskz_abs_epi64(k, a) simde_mm256_maskz_abs_epi64(k, a)
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m128i
+simde_mm_mask_add_epi8(simde__m128i src, simde__mmask16 k, simde__m128i a, simde__m128i b) {
+#if defined(SIMDE_X86_AVX512VL_NATIVE)
+  return _mm_mask_add_epi8(src, k, a, b);
+#else
+  return simde_mm_mask_mov_epi8(src, k, simde_mm_add_epi8(a, b));
+#endif
+}
+#if defined(SIMDE_X86_AVX512VL_ENABLE_NATIVE_ALIASES)
+#define _mm_mask_add_epi8(src, k, a, b) simde_mm_mask_add_epi8(src, k, a, b)
```

Comment:
Here too; if/elif/else preprocessor directives should increase the indentation of code inside of them (both other preprocessor macros and normal code).

I'll fix these when I merge, just something to look out for in the future.